internal/cmd/group_flag.go

```diff
@@ -243,7 +246,10 @@ func handleCSVFile(client *turso.Client, file, csvTableName string) (*turso.DBSe
 	}
 	defer csvFile.Close()
 
-	errs, invalid, _ := csvlint.Validate(csvFile, ',', false)
+	if len(csvSeparator) == 0 {
```

Comment:
⛏️: we usually just check if the string is empty.
```suggestion
	if csvSeparator == "" {
```

Nice defensive programming here!